gulpfile.js

```diff
@@ -51,12 +51,12 @@ gulp.task('dgeni', function() {
   }
 });
 
-gulp.task('markdown', function() {
-  return gulp.src('./docs/**/*.md')
-      .pipe(markdown())
-      .pipe(rename({extname: '.html'}))
-      .pipe(gulp.dest('dist/docs/'));
-});
+// gulp.task('markdown', function() {
+//   return gulp.src('./docs/**/*.md')
+//       .pipe(markdown())
+//       .pipe(rename({extname: '.html'}))
+//       .pipe(gulp.dest('dist/docs/'));
+// });
 
 
 // WATCH FILES FOR CHANGES
```

Comment:
I temporarily removed the markdown stuff as it is conflicting with the `marked` library in dgeni